brain_games/games/brain_calc_game.py

```diff
@@ -0,0 +1,36 @@
+#!usr/bin/env python3
+
+from random import randint, choice
+from typing import Tuple
+
+game_rules = "What is the result of the expression?"
+
+
+def multiplication(one, two) -> Tuple[str, str]:
+    game_task = f"{one} * {two}"
+    right_answer = str(one * two)
+    return game_task, right_answer
+
+
+def addition(one, two) -> Tuple[str, str]:
```

Comment:
please, additionally type the arguments as well
```suggestion
def addition(one: int, two: int) -> Tuple[str, str]:
```

Reply:
also, don't forget to update the project version with poetry